x5424/logger.go

```diff
@@ -26,10 +28,11 @@ type (
 
 // New returns a new Logger, level is a l5424 severity level string (unrecognized severity strings result
 // in a level of l5424.NoticeLvl). If w is nil os.Stdout is assumed.
-func New(level string, w io.Writer) *Logger {
+func New(level string, w io.Writer, color bool) *Logger {
```

Comment:
Let's keep the signature for New in tact (so it's backwards compatible) and make enabling the color a separate function.